docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -2,16 +2,16 @@
 Submitting contributions
 ========================
 
-We're always grateful for contributions to Django's code. Indeed, bug reports
-with associated contributions will get fixed *far* more quickly than those
-without a solution.
+We are always grateful for patches to Django's code.
```

Comment:
A preference for the word "contributions" over "patches" was just merged in https://github.com/django/django/pull/18264, so please pull the latest and make sure this is not reverted. Thanks